### ISSUE TITLE
Missing file for browser wasm in distributed npm package

### ISSUE THREAD

**Issue by 3cp** [parcel-bundler/source-map#99]
I tried to bundle `@parcel/source-map` in an app to run in browser, 

```
Cannot find module '../parcel_sourcemap_wasm/dist-web/parcel_sourcemap_wasm.js' from
'...../node_modules/@parcel/source-map/dist/wasm-bindings-web.js'
```

The file `parcel_sourcemap_wasm/dist-web/parcel_sourcemap_wasm.js` was not in the `@parcel/source-map` npm package current version v2.0.2.

It does contain the Rust source code in `parcel_sourcemap_wasm/src`. Is the compiled js code missed?

**Comment by mischnic** [comment]
For some reason, the wasm binary is missing from 2.0.1 and newer...

https://unpkg.com/browse/@parcel/source-map@2.0.0/parcel_sourcemap_wasm/dist-web/
https://unpkg.com/browse/@parcel/source-map@2.0.1/parcel_sourcemap_wasm/
https://unpkg.com/browse/@parcel/source-map@2.0.2/parcel_sourcemap_wasm/

**Comment by 3cp** [comment]
BTW, I am seeing over-all 5x (in nodejs env) performance improvement in my lib, by just replacing mozilla's source-map with `@parcel/source-map`.

Awesome work on this source map tool, guys!

**Comment by 3cp** [comment]
The latest v2.0.3 still has the same problem somehow.
https://unpkg.com/browse/@parcel/source-map@2.0.3/parcel_sourcemap_wasm/

**Comment by 3cp** [comment]
Sorry, I didn't realise this fix is not released yet.

**Comment by DeMoorJasper** [comment]
Released 2.0.4 which includes wasm files
